.gitignore

```diff
@@ -7,4 +7,5 @@
 *.code-workspace
 manifests/secrets-*.yaml
 config/*
-/allure-results
\ No newline at end of file
+/allure-results
+/scripts/*.log
```

Comment:
@mitri-slory Just a heads up I realized that the scripts log file was in the commit. Took it out and added a safeguard to prevent them in the future just for best practices reasons! (there hypothetically could be sensitive information in a log file).

Reply:
Ok. I understand.